src/tracer/src/daemon/initialization.rs

```diff
@@ -17,20 +17,22 @@ async fn get_db_client(init_args: &FinalizedInitArgs, config: &Config) -> LogWri
         println!("cli_config_args: {:?}", init_args);
         // if we pass --is-dev=false, we use the prod endpoint
         // if we pass --is-dev=true or don't pass the value, we use the dev endpoint
-        let forward_endpoint = &config.log_forward_endpoint_prod.as_ref().unwrap(); //TODO remove
-                                                                                    // if cli_config_args.is_dev.is_some() && cli_config_args.is_dev.unwrap().eq(&false) {
-                                                                                    //     println!(
-                                                                                    //         "Using prod endpoint: {}",
-                                                                                    //         &config.log_forward_endpoint_prod.as_ref().unwrap()
-                                                                                    //     );
-                                                                                    //     &config.log_forward_endpoint_prod.as_ref().unwrap()
-                                                                                    // } else {
-                                                                                    //     println!(
-                                                                                    //         "Using dev endpoint: {}",
-                                                                                    //         &config.log_forward_endpoint_dev.as_ref().unwrap()
-                                                                                    //     );
-                                                                                    //     &config.log_forward_endpoint_dev.as_ref().unwrap()
-                                                                                    // };
+        let forward_endpoint = &config.log_forward_endpoint_prod.as_ref().unwrap();
```

Comment:
can you revert these changes? i've worked on that on my new pr, so i removed the comment and put back the logic again, now `is_dev true` will point to clickhouse, in order to have both postgres and clikchouse to guarantee full availability